### ISSUE TITLE
Favicon not appearing in Safari's Reading List sidebar

### ISSUE THREAD

**Issue by hipwelljo** [RealFaviconGenerator/realfavicongenerator#390]
If you add your web page to Safari's Reading List, your favicon does not appear, instead it shows a generic monogram. (Testing with the latest release, Safari 11 on High Sierra.) 

I could be wrong, but I believe it's looking for a 64x64 (or 128x128 on Retina displays) size in the favicon.ico file. The FAQ says:

Why are certain PNG pictures missing?
64x64: Supported, but apparently not used by any browser or platform.
128x128: Chrome Web Store. Unless you turn your web application into a genuine Chrome application, this icon is not used.

So perhaps that's incorrect, Safari might be looking for those sizes.

**Comment by phbernard** [comment]
Thank you for reporting.

I ran the [favicon compatibility test](https://realfavicongenerator.net/favicon_compatibility_test) and I couldn't reproduce the issue on Safari 11.1.2 and High Sierra 10.13.6. Safari displays the 180x180 Apple Touch icon in the reading list. This makes sense: the Touch icon is a high resolution icon promoted by Apple, whereas `favicon.ico` is an old artifact from Microsoft, that I (as far as I remember) never saw in action on macOS.

However, when adding the compatibility test to the reading list, I noticed the icon appeared after a few seconds. At first, a default icon (a big "R"... probably the first letter of the domain name) was shown. This is a classic, observed for the mask/pinned sites icon, too. Maybe this is what you got?
